includes/definitions/audiocodes.yaml

```diff
@@ -0,0 +1,14 @@
+os: audiocodes
```

Comment:
audiocodes seems to be the manufacturers name. Is this the correct os name to use or should it be something like mediant?

Reply:
Yep, agreed ;) Looking at their products and OS families, it seems to be different with Audiocode ;) I really lack data for the moment to clearly decide. That's what we still have WIP in the title. 
Again, testdata from other devices would help us find out.